### DISCUSSION TITLE
Foxytab questions

### DISCUSSION THREAD

**Discussion by harryray33** [erosman/support#502] [Q&A]
Some questions:

What does capture do?
How does clean URL work?
Is it possible to keep the custom reload of a tab working when I restart Firefox, or do I have to set it each time?
Is it possible to choose where to bookmark all to?

Thanks.

**Comment by erosman** [comment]
> What does capture do?

As per FoxyTab Help:
Create and copy to clipboard, a data URI encoding an image of the visible area of the given tab. The default format of the resulting image is "png".


> How does clean URL work?

As per FoxyTab Help:
Experimental
Remove tracking/redirecting strings from Tab URLs

> Is it possible to keep the custom reload of a tab working when I restart Firefox, or do I have to set it each time?

Unfortunately not. The values are only kept temporarily.

> Is it possible to choose where to bookmark all to?

Unfortunately not.

**Comment by harryray33** [comment]
Thanks, any chance of including a persistent option to enable custom in a future release and an option of where to put the bookmarks?

**Comment by erosman** [comment]
If there is a popular demand for it, something can be worked out.
